AppCreationScripts/apps.json

```diff
@@ -15,7 +15,7 @@
       "x-ms-version": "2.0",
       "replyUrlsWithType": [
         {
-          "url": "urn:ietf:wg:oauth:2.0:oob",
+          "url": "https://login.microsoftonline.com/common/oauth2/nativeclient",
```

Comment:
Thanks @archieag 
we also need to change how the application is built (.WithDefaultRedirectUri) and update the README.md

Reply:
thanks, please merge this and send a PR for readme.